mapping-for-navigation/_posts/0000-01-09-modeling-interesections-for-map-navigation.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Modeling Intersections for Map Navigation
+---
+
+Capturing the various cases of Intersection modelling in OSM and the best practices to follow. The aim of this document is not go into the technicalities of traffic flow in a junction but rather to give a mapper clear cut instructions on how to model different intersections in OSM. Aiming to keep this document simple and easy to follow.
+
+### Case 1: When a transitioning dual carriageway intersects a bi-directional way. It has four ways in which the intersection can be designed 👇 
```

Comment:
Replace abstract descriptions with concrete examples so it’s clear when this guideline applies or doesn’t apply. For example, label the north–south street in each of the following diagrams as “Main Street” and the east–west street as “Broadway Avenue”:

> Main Street and Broadway Avenue both become divided at the place where they intersect. Draw the ways so that the intersection is between two single carriageways. That is, Main Street’s dual carriageway begins to the north of the intersection, and Broadway Avenue’s dual carriageway begins to the east of the intersection.